pkgs/development/python-modules/pacup/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  poetry2nix,
+  python,
+  fetchFromGitHub,
+}:
+poetry2nix.mkPoetryApplication rec {
+  pname = "pacup";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "pacstall";
+    repo = pname;
+    rev = version;
+    hash = "sha256-Hl/Gq/cZz4RGYKTuyDlrhATAUYEzKEuYIm0JdToN/ZY=";
+  };
+
+  projectDir = src;
+
+  # Propagating dependencies leads to issues downstream
+  # We've already patched poetry to prefer "vendored" dependencies
```

Comment:
Which we don't want because upstreams usually ping old things and we miss security fixes